src/redis_string.cc

```diff
@@ -19,23 +47,33 @@ rocksdb::Status String::getRawValue(const std::string &ns_key, std::string *raw_
   metadata.Decode(*raw_value);
   if (metadata.Expired()) {
     raw_value->clear();
-    return rocksdb::Status::NotFound("the key was expired");
+    return rocksdb::Status::NotFound(kErrMsgKeyExpired);
```

Comment:
As `mget`, redis also return `$-1\r\n`

Reply:
same with the previous comment, the NotFound error wouldn't throw to the user.